ush/machine/cheyenne.yaml

```diff
@@ -21,6 +21,8 @@ platform:
   PRE_TASK_CMDS: '{ ulimit -s unlimited; ulimit -a; }'
   TEST_EXTRN_MDL_SOURCE_BASEDIR: /glade/p/ral/jntp/UFS_SRW_App/develop/input_model_data
   TEST_PREGEN_BASEDIR: /glade/p/ral/jntp/UFS_SRW_App/develop/FV3LAM_pregen
+  TEST_ALT_EXTRN_MDL_SYSBASEDIR_ICS: /glade/p/ral/jntp/UFS_SRW_App/develop/dummy_FV3GFS_sys_dir
+  TEST_ALT_EXTRN_MDL_SYSBASEDIR_LBCS: /glade/p/ral/jntp/UFS_SRW_App/develop/dummy_FV3GFS_sys_dir
 task_run_fcst:
```

Comment:
What's the point of adding these directories and the "TEST_ALT"?

Reply:
@JeffBeck-NOAA Test cases such as `specify_EXTRN_MDL_SYSBASEDIR_ICS_LBCS` fail when I run the comprehensive test without it. Hera and Gaea had them before but the rest of the machines didn't so I added it to all of them in case such test cases are run there. The logic behind `TEST_ALT_*` seems to not be ideal, for example see this issue #364 that plans to change it.